### PR TITLE
Adding few lines for demonstration

### PR DESCRIPTION
**Pull request by Rathore4u**

### CODE REVIEW

**Review by RkKingsman — APPROVED** [review]
Reviewed

**Review by JayashreeYash — APPROVED** [review]

**Review by JayashreeYash — CHANGES_REQUESTED** [review]
Change

File: gitPracticeText.txt [hunk 1]
```diff
@@ -16,3 +16,10 @@ asa
 sasasas
 sasasa
 
+sasasas'
+asasas'a's'asas'a
```
Comment (Rathore4u) [inline gitPracticeText.txt:20]: this seems to be out of order 
this locator should be optimized by usign siblings 
example - "xpath "

File: gitPracticeText.txt [hunk 2]
```diff
@@ -16,3 +16,10 @@ asa
...
+
+
+a
+sasasa
```
Comment (Rathore4u) [inline gitPracticeText.txt:24]: Sassas should be used as SSSAASSAS 
Good work on remainnig methods

**Review by Rathore4u — COMMENTED** [review]
Please attend all PR comments

File: gitPracticeText.txt (hunk 1 above)
Comment (Rathore4u) [reply gitPracticeText.txt:20]: Thanks for the comment, 
But in my view this is the fact -- right

**Review by JayashreeYash — APPROVED** [review]

**Pull request merged**